hadoop-tools/hadoop-gcp/pom.xml

```diff
@@ -0,0 +1,614 @@
+                    <test.parallel.execution>false</test.parallel.execution>
+                    <!-- Propagate scale parameters -->
+                    <fs.gs.scale.test.enabled>${fs.gs.scale.test.enabled}</fs.gs.scale.test.enabled>
+                    <fs.gs.scale.test.huge.filesize>${fs.gs.scale.test.huge.filesize}</fs.gs.scale.test.huge.filesize>
+                    <fs.gs.scale.test.huge.huge.partitionsize>${fs.gs.scale.test.huge.partitionsize}</fs.gs.scale.test.huge.huge.partitionsize>
+                    <fs.gs.scale.test.timeout>${fs.gs.scale.test.timeout}</fs.gs.scale.test.timeout>
+                    <!-- Stream Type -->
+<!--                    <fs.gs.input.stream.type>${stream}</fs.gs.input.stream.type>-->
+                    <!-- are root tests enabled. Set to false when running parallel jobs on same bucket -->
+<!--                    <fs.gs.root.tests.enabled>${root.tests.enabled}</fs.gs.root.tests.enabled>-->
+                    <test.unique.fork.id>job-${job.id}</test.unique.fork.id>
+                  </systemPropertyVariables>
+                  <!-- Do a sequential run for tests that cannot handle -->
+                  <!-- parallel execution. -->
+                  <includes>
+                    <include>**/ITest*.java</include>
+                  </includes>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+    <profile>
+      <id>sequential-tests</id>
+      <activation>
+        <property>
+          <name>!parallel-tests</name>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-failsafe-plugin</artifactId>
+            <executions>
+              <execution>
+                <goals>
+                  <goal>integration-test</goal>
+                  <goal>verify</goal>
+                </goals>
+                <configuration>
+                  <trimStackTrace>false</trimStackTrace>
+                  <systemPropertyVariables>
+                    <!-- Propagate scale parameters -->
+                    <fs.gs.scale.test.enabled>${fs.gs.scale.test.enabled}</fs.gs.scale.test.enabled>
+                    <fs.gs.scale.test.huge.filesize>${fs.gs.scale.test.huge.filesize}</fs.gs.scale.test.huge.filesize>
+                    <fs.gs.scale.test.timeout>${fs.gs.scale.test.timeout}</fs.gs.scale.test.timeout>
+
+                    <test.unique.fork.id>job-${job.id}</test.unique.fork.id>
+                  </systemPropertyVariables>
+                  <forkedProcessTimeoutInSeconds>${fs.gs.scale.test.timeout}</forkedProcessTimeoutInSeconds>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+
+    <!-- Turn on scale tests-->
```

Comment:
Does the pom.xml have some stuff cloned from hadoop-tools/hadoop-aws/pom.xml that isn't relevant for this module?

Reply:
Yes, this is cloned from hadoop-aws/pom.xml. I kept it since we plan to have parity with other clients. I will remove this for now and add when we actually add those tests.